### PR TITLE
OCPBUGS-30111: Negative net interface name does not reduce queues

### PR DESCRIPTION
**Pull request by MarSik**
The internal regular expression has duplicated exclamation mark that breaks the pattern.

### CODE REVIEW

File: pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned_test.go [hunk 1]
```diff
@@ -411,15 +411,17 @@ var _ = Describe("Tuned", func() {
 					Expect(channelsRegex.MatchString(manifest)).To(BeTrue())
 				})
 				It("should set by negative interface name with reserved CPUs count", func() {
-					netDeviceName := "!ens5"
+					netDeviceName := "ens5"
+					netDeviceNameInverted := "!" + netDeviceName
+
 					//regex field should be: devices_udev_regex=^INTERFACE=(?!ens5)
 					devicesUdevRegex := "\\^INTERFACE=\\(\\?!" + strings.Replace(netDeviceName, "*", "\\.\\*", -1) + "\\)"
```
Comment (yanirq) [inline pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned_test.go:418]: should we still be using netDeviceName here ?    is it "pre stripped" ?

[the remark this answers is not in the document]
Comment (MarSik) [reply pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned_test.go:418]: YEs, this is a test, so netDeviceName is the clean variant so we can use it on line 418

**Pull request merged**